Product-listing/product-listing.css

```diff
@@ -0,0 +1,445 @@
+}
+
+button {
+    border-radius: 0;
+    box-shadow: var(--shadow-mild);
+}
+
+.button-icons {
+    font-size: 1rem;
+}
+
+.solid-secondary, .solid-primary {
+   flex-wrap: nowrap;
+   justify-content: center;
+   width:fit-content;
+}
+
+.solid-secondary {
+    justify-self: flex-end;
+}
+
+.solid-icon-only {
+    position: sticky;
+    bottom:1rem;
+    left:88vw;
+    font-size:1.2rem;
+    width:fit-content;
+    height:fit-content;
+    padding: 1rem 1.2rem;
+    color: var(--color-white);
+    border-radius: var(--circle-border);
+    z-index:8;
+    margin-top: 1rem;
+    display:none;
+}
+
+
+.action-tab .actions{
+    display:grid;
+    grid-template-columns: 1fr 40%;
+}
+
+footer {
+    position: sticky;
+    background-color: var(--color-secondary);
+    color: var(--color-white);
+    padding: 2rem;
+    flex-wrap: wrap;
+}
+
+footer h6 {
+    padding: 1rem 0;
+}
+
+li {
+    list-style: none;
+}
+
+.social-links {
+    gap: 1rem;
+    padding-top: 1rem;
+}
+
+.brand-info .links {
+    font-size: 1.7rem;
+}
+
+.sitemap .links {
+    font-size: 1.3rem;
+}
+
+.sitemap {
+   gap: 1rem
+}
+
+footer input {
+    color: var(--color-white);
+}
+
+footer input::placeholder {
+    color: var(--color-faint-white);
+}
+
+.solid-secondary {
+    color: var(--color-white)
+}
+
+footer p {
+    font-size: 1.2rem;
+}
+
+.fixed {
+    position: fixed;
+    bottom: 0;
+    left: 0;
+    right: 0;
+}
+
+
+@media (max-width:900px) {
+    html {
+        font-size:15px;
+    }
+}
+
+@media (max-width: 800px) {
+    
+    .main-wrapper.grid-container {
+        display:flex;
+        flex-direction: column;
+        position:relative;
+        padding:0;
+    }
+
+    .product-display {
+        border:none
+    } 
+
+    .filter {
+        padding: 1rem 3rem;
+        width: 100%;
+    }
+
+    .filter .heading {
+        justify-content:flex-end;
+    }
+
+    .responsive-header {
+        display:none;
+    }
+
+    .filter-film {
+        content:"";
+        backdrop-filter: blur(10px);
+        position: fixed;
+        top:0;
+        bottom:0;
+        left:0;
+        right:0;
+        z-index: 3;
+        display:none;
+    }
+
+    .solid-icon-only {
+        display:block;
+    }
+
+    .filter-dropdown {
+        display:block;
+        z-index: 5;
+    }
+
+    .filter-contents {
+        overflow-y: auto;
+       display: none;
+       z-index: 5;
+    }
+
+    .filter {
+        display:none;
+    }
+
+    .hamburger-nav {
+        display:block;
+    }
+}
+
+@media (max-width: 600px) {
+    .contact-you {
+        margin-top: 2rem;
+    }
+
+    .card {
+        width: 100%;
+    }
+
+    footer {
+        padding:2rem 1rem;
+    }
+
+}
+
+@media (max-width: 480px) {
+
+    html {
+        font-size: 13px;
+    }
+
+    nav {
+        padding: 2rem;
+    }
+
+    .input-box, .brand-name,.brand-info, .sitemap, .contact-you {
+        flex-grow: 1;
+    }
+
+    /* .product-display {
+        grid-template-columns: 1fr;
+    } */
+
```

Comment:
Can remove these commented lines of code

Reply:
Thanks for pointing this out